content/modules/rest/pages/user-session-api.adoc

```diff
@@ -0,0 +1,141 @@
+= User Session API
+
+The User Session API provides information that is influenced by the current user session. It consists of the following two endpoints:
```

Comment:
Let's make it simpler and avoid tautology:
"The User Session API consists of the following two endpoints:"